src/LogFileReader.cpp

```diff
@@ -75,11 +80,25 @@ bool LogFileReader::readAll(const QString& fileName)
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
     // TODO implement this first
-
+    QStringList lines = line.split(',');
+    bool ok;
+    //I have to split up the time and round up the second into an integer in order for fromString to work
+    //Not sure how to use fromString with 3 decimals in second
+    QStringList timelist = lines[0].split(':');
+    double second = timelist[2].toDouble(&ok);
+    second = round(second);
+    timelist[2] = QString::number(second);
+    lines[0]= timelist.join(":");
+    QTime time = QTime::fromString(lines[0],"hh:mm:s");
```

Comment:
Similarly to the delimiter, `"hh:mm:s"` can be put into the namespace as a const QString as well. 

Also, the format of the times in batData.csv is actually `hh:mm:ss.zzz`.